src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -1385,6 +1385,12 @@ bool LibraryCallKit::inline_vector_reduction() {
   int num_elem = vlen->get_con();
   int opc  = VectorSupport::vop2ideal(opr->get_con(), elem_bt);
   int sopc = ReductionNode::opcode(opc, elem_bt);
+  if (sopc == opc) {
```

Comment:
Can you merge this with the following `if`?

Reply:
Yes, i merged them.